cmake/scripts/rbpi/ArchSetup.cmake

```diff
@@ -13,9 +13,12 @@ else()
   if(CPU STREQUAL arm1176jzf-s)
     set(ARCH arm)
     set(NEON False)
+    set(NEON_FLAGS "-mcpu=arm1176jzf-s -mtune=arm1176jzf-s -mfloat-abi=hard -mfpu=vfp")
   elseif(CPU MATCHES "cortex-a7" OR CPU MATCHES "cortex-a53")
     set(ARCH arm)
     set(NEON True)
+    set(NEON_FLAGS "-fPIC -mcpu=cortex-a7 -mfloat-abi=hard -mfpu=neon-vfpv4 -mvectorize-with-neon-quad")
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lpthread")
```

Comment:
As explained before, these flags copy the existing ones from https://github.com/xbmc/xbmc/blob/master/tools/depends/configure.ac#L394-L411 for both platforms. 
If you look a bit above (on line 16) you'll see that there are different flags, more specific to RPi 0/1 CPU.

I'm not sure if this answers your question, but if not please let me know.